tests/test_eradiate/conftest.py

```diff
@@ -59,3 +59,10 @@ def ert_seed_state():
     from eradiate.rng import SeedState
 
     return SeedState(0)
+
+
+@pytest.fixture(scope="session")
+def session_timestamp():
+    from datetime import datetime
+
+    return datetime.now()
```

Comment:
Maybe this should go one level up? Or do you want this fixture to be restricted to non-plugin tests?